src/clib/lib/job_queue/lsf_driver.cpp

```diff
@@ -735,24 +735,41 @@ void lsf_driver_kill_job(void *_driver, void *_job) {
         char **argv = (char **)calloc(2, sizeof *argv);
         CHECK_ALLOC(argv);
         argv[0] = driver->remote_lsf_server;
-        argv[1] = saprintf("%s %s %s", driver->bkill_cmd, "-s SIGKILL",
+        argv[1] = saprintf("%s %s %s", driver->bkill_cmd, "-s SIGTERM",
                            job->lsf_jobnr_char);
-
-        spawn_blocking(driver->rsh_cmd, 2, (const char **)argv, NULL, NULL);
-
         free(argv[1]);
         free(argv);
+
+        char **argv2 = (char **)calloc(2, sizeof *argv2);
```

Comment:
Maybe spend some keystrokes on saying what `argv2` is for, like "argv_second_kill"

Reply:
A refactor of this area of code is pending: https://github.com/equinor/ert/pull/7515/files

Technically, it's enough to remove the `char **` from `char **argv2`. But it's possible to improve the readability of this code quite easily by writing C++, so I recommend we ignore this until the aforementioned PR.